.travis.yml

```diff
@@ -50,11 +51,15 @@ jobs:
     - name: cephcsi
       script:
         - make cephcsi ||  travis_terminate 1;
+        - make image-cephcsi || travis_terminate 1;
         - sudo scripts/minikube.sh up || travis_terminate 1;
         # pull docker images to speed up e2e
         - scripts/minikube.sh cephcsi
         - scripts/minikube.sh k8s-sidecar
         - "sudo chown -R travis: $HOME/.minikube /usr/local/bin/kubectl"
+        # functional tests
+        - make func-test TESTOPTIONS='--rook-version=v1.0.0 \
```

Comment:
`v1.1.0` ?

Reply:
This is the current stable version in rook, so using stable release

Reply:
`v1.0.1` is the latest stable as of today

Reply:
@travisn Thanks, updated to `v1.0.1`